src/plugins/discover/public/application/components/default_discover_table/default_discover_table.tsx

```diff
@@ -118,10 +120,14 @@ const DefaultDiscoverTableUI = ({
           if (entries[0].isIntersecting) {
             // Load another batch of rows, some immediately and some lazily
             setRenderedRowCount((prevRowCount) => prevRowCount + INFINITE_SCROLLED_PAGE_SIZE);
-            setDesiredRowCount((prevRowCount) => prevRowCount + 5 * INFINITE_SCROLLED_PAGE_SIZE);
+            setDesiredRowCount((prevRowCount) => prevRowCount + DESIRED_ROWS_LOOKAHEAD);
           }
         },
-        { threshold: 1.0 }
+        {
+          // Important that 0 < threshold < 1, since there OSD application div has a transparent
+          // fade at the bottom which causes the sentinel element to sometimes not be 100% visible
+          threshold: 0.5,
```

Comment:
why not use .001 so the observer will trigger loading as soon as the user scrolls there?

Reply:
The progress bar is 2 pixels wide, with 50% opacity that makes any threshold lower than 25% redundant... I'll set it to 0.1